app/services/flowcommerce_spree/order_sync.rb

```diff
@@ -168,6 +169,7 @@ def fetch_session_id
       @order.flow_data['session_expires_at'] = session_expire_at
 
       if session.respond_to?(:[])
+        cookies['_f60_session'] = { value: current_session_id, domain: '.mejuri.com' }
```

Comment:
move this line before `      if session.respond_to?(:[])` and change it to check if cookies is a hash as well:

```suggestion
        cookies['_f60_session'] = { value: current_session_id, domain: '.mejuri.com' } if cookies.respond_to?(:[])
```